contracts/Governance/AccessControlledV5.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.5.16;
+
+import "./Utils/InitializableV5.sol";
+import "./Utils/Ownable2StepUpgradeableV5.sol";
+import "./IAccessControlManagerV5.sol";
+
+/**
+ * @title Venus Access Control Contract.
+ * @dev This contract is helper between access control manager and actual contract
+ * This contract further inherited by other contract to integrate access controlled mechanism
+ * It provides initialise methods and verifying access methods
+ */
+
+contract AccessControlledV5 is InitializableV5, Ownable2StepUpgradeableV5 {
```

Comment:
I'm thinking whether we really need `Ownable2StepUpgradeableV5` here – AFAIU the admins of the vaults are set already, so this new `owner` field would live in parallel with the existing one. That's why, AFAIU, you needed a new `_setAccessControl` method in XVSVault rather than used `setAccessControlManager` defined in this contracts – it's because the owners of AccessControlledV5 and vaults are different. I understand the intent here, and I think making use of common base contracts is a good thing in general, but IMO having two distinct owners of the same contract will cause confusion.